src/test/groovy/net/fabricmc/loom/test/integration/LegacyProjectTest.groovy

```diff
@@ -86,8 +86,41 @@ class LegacyProjectTest extends Specification implements GradleProjectTestTrait
 		'1.3.2'			| _
 	}
 
+	// TODO: need intermediaries for this test to succeed
```

Comment:
This needs doing.

Reply:
If I have a mapping file in `/source/test/recources/`, what would be the way to go about writing the test so that it uses it?